app/Http/Requests/API/FormRequest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Requests\API;
+
+use Illuminate\Contracts\Validation\Validator;
+use Illuminate\Foundation\Http\FormRequest as LaravelFormRequest;
+use Illuminate\Http\Exceptions\HttpResponseException;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Validation\ValidationException;
+
+abstract class FormRequest extends LaravelFormRequest
```

Comment:
This solution is not needed as discussed. If you set the `Accept` header to `application/json` it will automatically return the form validation errors in a JSON formt.